README.md

```diff
@@ -151,54 +155,33 @@ Release [v2.0.4](https://github.com/vechain/thor/releases/tag/v2.0.4) changed th
 
 ## Explorers
 
-* [VeChain Explorer (Official)](https://explore.vechain.org)
-* [VeChainStats](https://vechainstats.com/)
+* [Vechain Explorer (Official)](https://explore.vechain.org)
+* [VechainStats](https://vechainstats.com/)
 * [Insight](https://insight.vecha.in/)
 
 ## Testnet faucet
 
-* [faucet.vecha.in](https://faucet.vecha.in) by *VeChain Foundation*
+* [faucet.vecha.in](https://faucet.vecha.in) by *Vechain Foundation*
 
 ## API
 
 Once `thor` has started, the online *OpenAPI* doc can be accessed in your browser. e.g. [http://localhost:8669/](http://localhost:8669) by default.
 
-[![Thorest](https://raw.githubusercontent.com/vechain/thor/master/thorest.png)](http://localhost:8669/)
+[![Thorest](./thorest.png)](http://localhost:8669/)
```

Comment:
Keep the full URL, it's for other sources displaying the image, for example https://hub.docker.com/r/vechain/thor